sonic_platform_base/module_base.py

```diff
@@ -372,6 +374,95 @@ def get_thermal(self, index):
 
         return thermal
 
+    ##############################################
+    # Voltage Sensor methods
+    ##############################################
+
+    def get_num_vsensors(self):
+        """
+        Retrieves the number of voltage sensors available on this module
+
+        Returns:
+            An integer, the number of voltage sensors available on this module
+        """
+        return len(self._vsensor_list)
+
+    def get_all_vsensors(self):
+        """
+        Retrieves all voltage sensors available on this module
+
+        Returns:
+            A list of objects derived from VsensorBase representing all voltage
+            sensors available on this module
+        """
+        return self._vsensor_list
+
+    def get_vsensor(self, index):
+        """
+        Retrieves voltage sensor unit represented by (0-based) index <index>
+
+        Args:
+            index: An integer, the index (0-based) of the voltage sensor to
+            retrieve
+
+        Returns:
+            An object dervied from VsensorBase representing the specified voltage
+            sensor
+        """
+        vsensor = None
+
+        try:
+            vsensor = self._vsensor_list[index]
+        except IndexError:
+            sys.stderr.write("Voltage sensor index {} out of range (0-{})\n".format(
+                             index, len(self._vsensor_list)-1))
+
+        return vsensor
+
+    ##############################################
+    # Current sensor methods
+    ##############################################
+
+    def get_num_Isensors(self):
+        """
+        Retrieves the number of Current sensors available on this module
```

Comment:
'Current' should be lowercase throughout this docstring

Reply:
ack